packages/grid/data-grid/src/tests/layout.DataGrid.test.tsx

```diff
@@ -185,6 +183,25 @@ describe('<DataGrid /> - Layout & Warnings', () => {
         }).toWarnDev(["You are calling getValue('age') but the column `age` is not defined"]);
         expect(getColumnValues()).to.deep.equal(['1', '2']);
       });
+
+      it('should have a stable height if the parent container has no intrinsic height', () => {
```

Comment:
Not sure if this is the best way to do it but without the check for the `console.warn` the test fails because if the parent doesn't have height the gird throws a warning.

Reply:
Why did you remove the `p`? Does it still reproduce the issue if you add this test on HEAD and remove style.height = 0? I bet it doesn't because it's a key aspect of the infinite height growth. 

Also using clock.tick with a sleep looks wrong. It can only be one or the other. Considering the test pass, I would bet that clock.tick is a dead logic.

Reply:
Then it is my fault for not understanding the issue. I thought that it grows infinitely when the parent height is 0. My bad.